cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -535,7 +535,7 @@ class QueueNetworkingSchema(BaseNetworkingSchema):
     subnet_ids = fields.List(
         fields.Str(validate=get_field_validator("subnet_id")),
         required=True,
-        validate=validate.Length(equal=1),  # FIXME Add multi-subnet support for Awsbatch
+        validate=validate.Length(equal=1),  # FIXME Add multi-subnet support for AwsBatch
```

Comment:
we can probably remove this FIXME cause it's more like a new feature than something to fix

Reply:
Removed